tests/api_fastapi/core_api/routes/public/test_xcom.py

```diff
@@ -203,3 +208,279 @@ def test_custom_xcom_deserialize(
         else:
             assert response.status_code == 200
             assert response.json()["value"] == expected_status_or_value
+
+
+class TestGetXComEntries(TestXComEndpoint):
+    @pytest.fixture(autouse=True)
+    def setup(self) -> None:
+        self.clear_db()
```

Comment:
We shouldn't be needing this because setup and teardown are defined in class: `TestXComEndpoint` which `TestGetXComEntries` inherits from

Reply:
This was to override the original setup to not pre-include creating a dag run, since its handled slightly differently within `TestGetXComEntries._create_xcom_entries`